docs/content/preview/develop/pg15-features.md

```diff
@@ -0,0 +1,184 @@
+| Store results in memory when the inner side of a nested-loop join is small. |
+
+| [Disk-based hash aggregation](../../architecture/query-layer/#optimizations)
+| Opt for disk-based aggregation when hash table grows beyond `work_mem`. |
+
+{{%/table%}}
+
+### Observability
+
+{{%table%}}
+
+| Feature | Description |
+| --------| ----------- |
+
+| [Query ID](../../architecture/query-layer/#query-id)
+| Unique query ID to track a query across pg_stat_activity, EXPLAIN VERBOSE, and pg_stat_statements. |
+
+| [Stats on planning times](../../explore/query-1-performance/pg-stat-statements/)
+| pg_stat_statements can now track the planning time of statements. |
+
+| [Granular stats reset](../../explore/query-1-performance/pg-stat-statements/#reset-statistics)
+| Resetting statistics via `pg_stat_statements_reset` is now granular at user, database, and query levels. |
+
+| [Sampled logging](../../explore/observability/logging/#log-management)
+| Log a fraction of the statements rather than all statements. |
+
+{{%/table%}}
+
+### Security
+
+{{%table%}}
+
+| Feature | Description |
+| --------| ----------- |
+
+| [Stricter public schema](../../api/ysql/the-sql-language/statements/dcl_create_user/#public-schema-privileges)
+| CREATE Privilege for PUBLIC Role has been revoked ensuring tighter control over object creation, reducing the risk of accidental or malicious interference in shared schemas.|
+
+| [Simplified&nbsp;privilege management](../../api/ysql/the-sql-language/statements/dcl_grant/#predefined-roles)
+| New roles have been added to streamline permission assignments by grouping commonly needed privileges for read and write operations. |
+
+{{%/table%}}
+
+### Coming soon
+
+The following PG15 features are not yet implemented but are planned for the future.
+
+{{%table%}}
+
+| Feature | Description |
+| --------| ----------- |
+
+| [Extended statistics](https://www.postgresql.org/docs/15/planner-stats.html#PLANNER-STATS-EXTENDED)
+| Gather additional statistics using the [CREATE STATISTICS](https://www.postgresql.org/docs/15/sql-createstatistics.html) command. |
+
+| [Merge command](https://www.postgresql.org/docs/current/sql-merge.html)
+| INSERT, UPDATE or DELETE in one statement. |
+
+| [Scram authentication as default](https://www.postgresql.org/docs/15/auth-password.html)
+| Scram authentication is [supported](../../secure/authentication/password-authentication/#scram-sha-256) in YugabyteDB but still has md5 as default authentication method. |
+
+| [Nondeterministic collations](https://www.postgresql.org/docs/12/collation.html#COLLATION-NONDETERMINISTIC)
+| Consider strings to be equal even if they consist of different bytes, for example, case-insensitive, or accent-insensitive comparisons. |
+
+{{%/table%}}
+
+## Upgrading
+
+When upgrading from PoistgreSQL 11-compatible versions of YugabyteDB (prior to v2.25) to a PostgreSQL 15 compatible version, keep in mind there are changes in some behaviors between PostgreSQL 11 and PostgreSQL 15.
```

Comment:
/s/PoistgreSQL/PostgreSQL

Reply:
May be rephrase as following : 


`When YugabyteDB cluster is upgraded from from PostgreSQL 11-compatible versions(2024.2 or prior releases) to PostgreSQL 15 compatible version (2.25.0 or later releases) then due to Open Source PostgreSQL changes  there is specific behavior changes in some of the operations which are listed below.`